libs/filamat/src/UibGenerator.cpp

```diff
@@ -71,6 +71,7 @@ UniformInterfaceBlock const& UibGenerator::getPerViewUib() noexcept  {
             // camera
             .add("exposure",                1, UniformInterfaceBlock::Type::FLOAT, Precision::HIGH) // high precision to work around #3602 (qualcomm)
             .add("ev100",                   1, UniformInterfaceBlock::Type::FLOAT)
+            .add("needsAlphaChannel",       1, UniformInterfaceBlock::Type::FLOAT)
```

Comment:
Continuing the previous:
```suggestion
            .add("needsAlphaChannel",       1, UniformInterfaceBlock::Type::FLOAT)
            .add("padding3",                1, UniformInterfaceBlock::Type::FLOAT3)
```